infertrade/algos/community/allocations.py

```diff
@@ -25,24 +25,24 @@
 from infertrade.PandasEnum import PandasEnum
 
 
-def fifty_fifty(dataframe) -> pd.DataFrame:
+def fifty_fifty(dataframe: pd.DataFrame) -> pd.DataFrame:
     """Allocates 50% of strategy budget to asset, 50% to cash."""
-    dataframe["allocation"] = 0.5
+    dataframe[PandasEnum.ALLOCATION.value] = 0.5
     return dataframe
```

Comment:
This is good in general. Only reason I didn't convert this earlier is so that those new to Python would potentially see a simpler function that didn't require references to Enums and awareness of argument hints. However converting over does make it more consistent with the rest of the code! Happy to follow your thoughts on this.

Reply:
@ta-oliver Oh, I completely agree. I'll revert this.

Reply:
Argument hints I think are helpful, I just know some Python programmers haven't encountered them. I generally think they help to understand what is being passed to the function.

Reply:
Oh I forgot about the hints. I'll revert those too. I think it is a good idea to have at least one function that is simpler.

Reply:
Reverted the argument hint. The return type was already present in previous versions of the file.